Src/BootCamp.Chapter1/ArrayOperations.cs

```diff
@@ -2,14 +2,42 @@
 {
     public static class ArrayOperations
     {
+
+        static bool IsArraySorted(int[] array)
+        {
+            for (int i = 0; i < array.Length + 1; i++)
+            {
+                //This to stop IndexOutOfRangeException
+                if (i + 1 >= array.Length) break;
+
+                if (array[i] > array[i + 1]) return false;
+            }
+            return true;
+        }
         /// <summary>
         /// Sort the array in ascending order.
         /// If array empty or null- don't do anything.
         /// </summary>
         /// <param name="array">Input array in a random order.</param>
         public static void Sort(int[] array)
         {
-            // ToDo: implement.
+            if (array == null || array.Length == 0) return;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add curly braces around the nested statement(s) in this 'if' block.](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=4904050)